crates/reward/src/tests.rs

```diff
@@ -0,0 +1,53 @@
+/// Tests for Reward
+use crate::mock::*;
+use crate::Rewards;
+use frame_support::assert_ok;
+
+// type Event = crate::Event<Test>;
```

Comment:
what will happen with rewards that cannot be divided easily? Say Alice and Bob have 50 stake each and rewards are 451. I guess everybody should get 225 or one gets 226 and one gets 225. Please add a test case for this.